### ISSUE TITLE
how to use load a model checkpoint using INIT_CHECKPOINT

### ISSUE THREAD

**Issue by schelv** [google-research/bert#330]
Hi,

I want to find out which of the saved checkpoints works the best on the validation set.
However I cannot load a specific checkpoint using `INIT_CHECKPOINT `.
Instead it loads the latest model.
Issue #110 provides a fix for this.
Is it possible to load a specific checkpoint without changing the checkpoint file?
Am I using `INIT_CHECKPOINT ` wrong?

~~~~  INIT_CHECKPOINT = os.path.join(BERT_FINETUNED_DIR, 'model.ckpt-0')

train_examples = None
num_train_steps = None

model_fn = run_classifier.model_fn_builder(
    bert_config=modeling.BertConfig.from_json_file(CONFIG_FILE),
    num_labels=len(label_list),
    init_checkpoint=INIT_CHECKPOINT,
    learning_rate=LEARNING_RATE,
    num_train_steps=num_train_steps,
    num_warmup_steps=num_warmup_steps,
    use_tpu=True,
    use_one_hot_embeddings=True)

estimator = tf.contrib.tpu.TPUEstimator(
    use_tpu=True,
    model_fn=model_fn,
    config=run_config,
    train_batch_size=TRAIN_BATCH_SIZE,
    predict_batch_size=PREDICT_BATCH_SIZE,
    eval_batch_size=EVAL_BATCH_SIZE) 
~~~~
